webapp.yml

```diff
@@ -0,0 +1,71 @@
+# Description
+# ===========
+# This playbook create an Azure Linux Web app with Java container Tomcat 8.5,
+# then build a java hello world app, then ftp the war file to the Web app.
+#
+# Dependencies
+# ============
+# git
+# maven
+- hosts: localhost
+  connection: local
+  vars:
+    app_name: ansible_app1
+    location: eastus
+    linux_plan_name: linux_appplan1
+    repo_url: https://github.com/yungezz/helloworld.git
+    workspace: ~/helloworld
+
+  roles:
+    - Azure.azure_preview_modules
```

Comment:
it should be mentioned somewhere in prerequisites how to install preview modules

Reply:
it's in  repo readme, it's not appropriate to put in each sample